sd-core/src/language/mod.rs

```diff
@@ -66,7 +66,7 @@ pub trait Language {
     type Var: Syntax + Fresh + From<Self::Symbol> + WithType;
     type Addr: Syntax;
     type BlockAddr: Syntax;
-    type VarDef: Syntax + GetVar<Self::Var>;
```

Comment:
Why do we need this change? is the colour stored as a type?

Reply:
It's because I changed `sd-lang` to only let you define the colour of variables when a variable is defined